src/test/java/test/listeners/github2385/TestMultiInherit.java

```diff
@@ -0,0 +1,14 @@
+package test.listeners.github2385;
+
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertTrue;
+
+public class TestMultiInherit implements ITestClassInterface, ITestInterface{
+
+    @Test
+    public void testMultiInherit() {
+        assertTrue(TestListener.listenerMethodInvoked);
```

Comment:
except for some specific cases, assertions are often useless in samples

Reply:
assert in sample are useless, so I deleted them